test/asyncpool.test.ts

```diff
@@ -91,10 +91,10 @@ suite("AsyncPool Tests", () => {
         let pool = new AsyncPool(8);
         let arr: number[] = [];
         let arr2: number[] = [];
-        for (let i = 0; i < 30000; i++) {
+        for (let i = 0; i < 300; i++) {
             arr.push(i);
         }
-        for (let i = 0; i < 30000; i++) {
+        for (let i = 0; i < 300; i++) {
```

Comment:
It looks like some iterations were shrunk to 1/10, some to 1/2, and some to 1/100. 
Was this from experimentation or heuristic that guided this?

Reply:
Just getting them all to a reasonable time.  Some have built-in delays.